packages/lesswrong/server/scripts/mergeAccounts.js

```diff
@@ -0,0 +1,149 @@
+/* global Vulcan */
+import Users from 'meteor/vulcan:users';
+import { editMutation, getCollection, Utils } from 'meteor/vulcan:core';
+import { Revisions } from '../../lib/index';
+import { editableCollectionsFields } from '../../lib/editor/make_editable'
+import ReadStatuses from '../../lib/collections/readStatus/collection';
+import { Votes } from '../../lib/collections/votes/index';
+import { Conversations } from '../../lib/collections/conversations/collection'
+
+
+
+const transferOwnership = async ({documentId, targetUserId, collection, fieldName = "userId"}) => {
+  await editMutation({
+    collection,
+    documentId,
+    set: {[fieldName]: targetUserId},
+    unset: {},
+    validate: false,
+  })
+}
+
+const transferCollection = async ({sourceUserId, targetUserId, collectionName, fieldName = "userId"}) => {
+  const collection = getCollection(collectionName)
+  const documents = await collection.find({[fieldName]: sourceUserId}).fetch()
+  // eslint-disable-next-line no-console
+  console.log(`Transferring ${documents.length} documents in collection ${collectionName}`)
+  for (const doc of documents) {
+    await transferOwnership({documentId: doc._id, targetUserId, collection, fieldName})
+    // Transfer ownership of all revisions and denormalized references for editable fields
+    const editableFieldNames = editableCollectionsFields[collectionName]
+    if (editableFieldNames?.length) {
+      editableFieldNames.forEach((editableFieldName) => {
+        transferEditableField({documentId: doc._id, targetUserId, collection, fieldName: editableFieldName})
+      })
+    }
+  }
+  documents.forEach((doc) => {
+    
+  })
+}
+
+const transferEditableField = ({documentId, targetUserId, collection, fieldName = "contents"}) => {
+  // Update the denormalized revision on the document
+  editMutation({
+    collection,
+    documentId,
+    set: {[`${fieldName}.userId`]: targetUserId},
+    unset: {},
+    validate: false
+  })
+  // Update the revisions themselves
+  Revisions.update({ documentId, fieldName }, {$set: {userId: targetUserId}}, { multi: true })
+}
+
+const mergeReadStatusForPost = ({sourceUserId, targetUserId, postId}) => {
+  const sourceUserStatus = ReadStatuses.findOne({userId: sourceUserId, postId})
+  const targetUserStatus = ReadStatuses.findOne({userId: targetUserId, postId})
+  const readStatus = !!(sourceUserStatus?.isRead || targetUserStatus?.isRead)
```

Comment:
Should be the `isRead` from whichever of these has a newer `lastUpdated`, to interact correctly with mark-as-unread.